bootstrap/config/base/kfctl_default.yaml

```diff
@@ -25,7 +25,12 @@ spec:
   - centraldashboard
   - admission-webhook  
   - jupyter-web-app
-  - katib
+  - katib-db
+  - katib-manager
+  - katib-controller
+  - katib-ui
+  - metrics-collector
+  - suggestion
```

Comment:
What is "suggestion"?

Reply:
Katib algorithm suggestions (grid, random, bayesian, etc)